spatialmath/point.go

```diff
@@ -11,34 +11,34 @@ import (
 
 // PointCreator implements the GeometryCreator interface for point structs.
 type pointCreator struct {
-	offset Pose
+	offset r3.Vector // this needs to be changed
 	label  string
 }
 
 // point is a collision geometry that represents a single point in 3D space that occupies no geometry.
 type point struct {
-	pose  Pose
+	pose  r3.Vector // remove this field
```

Comment:
`pose` should be re-named because the type is not longer `Pose`.
Any suggestions?